pkgs/development/python-modules/debtcollector/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pbr
+, six
+, wrapt
+}:
+let 
+  pyModuleDeps = [
+    pbr
+    six
+    wrapt
+  ];
+in
+buildPythonPackage rec {
+  pname = "debtcollector";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    pname = "debtcollector";
+    inherit version;
+    sha256 = "0ab8ic9bk644i8134z044kpcf6pvawqf4rq4xgv1p11msbs82ybq";
+  };
+  
+  buildInputs = pyModuleDeps;
+  propagatedBuildInputs = pyModuleDeps;
```

Comment:
Why are both of these set? Anything in propagatedBuildInputs is available at build time.

Reply:
Fixed.

Reply:
It would be better to not have the separate pyModuleDeps variable now that it is no longer needed.